ROMFS/px4fmu_common/init.d/4040_reaper

```diff
@@ -2,7 +2,7 @@
 #
 # @name Reaper 500 Quad
 #
-# @type Quadrotor x
```

Comment:
As per doc PR, suggest we add the motor order information for this quad here too - otherwise the whole "common outputs" section will be empty for the quadrotor h case.